llvm/lib/Transforms/Vectorize/VPlanUtils.h

```diff
@@ -25,6 +25,9 @@ bool onlyFirstLaneUsed(const VPValue *Def);
 /// Returns true if only the first part of \p Def is used.
 bool onlyFirstPartUsed(const VPValue *Def);
 
+/// Returns true if \p Def only uses scalars.
```

Comment:
The comment is the wrong way round I think. It is checking if all users of Def only use scalar values from Def.